### ISSUE TITLE
Terminal command inoperable when using minikube

### ISSUE THREAD

**Issue by cmeyertons** [Azure/vscode-kubernetes-tools#465]
### Description
When I use the `Terminal` command on a minikube pod, the terminal does not properly load.  I can see `/ #` but cannot interact with it at all.  I am currently using `Powershell` as my terminal, but ruled that out as a cause by trying `cmd` and `git-bash`.

I have confirmed in my environment that this is isolated to `minikube` - I can terminal just fine into my AKS pods.

Looking through the issue, i thought this might be related to #333 so I tried to set the following workspace settings w/ no luck:

```javascript
{
    "terminal.integrated.env.windows": {
        "DOCKER_CERT_PATH": "C:\\Users\\carlm\\.minikube\\certs",
        "DOCKER_HOST":"tcp://192.168.86.221:2376",
        "DOCKER_TLS_VERIFY": "1"
    }
}
```

### Steps to reproduce:

1. download minikube
1. set up hyperv virtual switch
1. `minikube start --vm-driver hyperv --hyperv-virtual-switch minikube`
1. go to Nodes -> right click one of  the system pods -> Terminal
1. results:

![image](https://user-images.githubusercontent.com/19917677/52412748-4e1cee00-2aa5-11e9-957e-4cdb72a4039f.png)

OS: windows 10
VM Driver: hyperv

**Comment by itowlson** [comment]
Can you (from a Powershell window) run an interactive session within a Minikube pod from the `kubectl` CLI?  E.g.

`kubectl exec -it <podname> -- sh`

**Comment by cmeyertons** [comment]
Yes! From powershell or from integrated Code terminal. I resorted to writing some “remote into xxx” VS code tasks that accomplish this as a workaround - the interactive shell produced by the task has some really weird quirks as well

**Comment by itowlson** [comment]
Hmm, it's working for me (well, after some shenanigans getting Minikube to work _at all_), at least in an application pod (e.g. an `nginx` container).  But you mentioned that you were terminal-ing into a system pod.  If I try that, even from the command line, I get:

```
C:\>kubectl exec -it coredns-86c58d9df4-4ccfc -- sh
OCI runtime exec failed: exec failed: container_linux.go:344: starting container process caused "exec: \"sh\": executable file not found in $PATH": unknown
command terminated with exit code 126
```

If I try to use the VS Code Terminal command on this pod, then it briefly flashes a terminal window which (presumably displays the same message and) then exits.

1. Can you try deploying `nginx` and see if you can terminal into that pod?
2. Can you provide more information about exactly which system pod you are terminal-ing into (e.g. CoreDNS, etcd, etc.)?
3. You mention that you get the to system pod by expanding the Nodes folder, but I don't see any pods under there, only under Workloads.  Could you confirm this is what you're doing please, and maybe provide a screenshot to show what you see?

Thanks!

**Comment by cmeyertons** [comment]
sorry i'm not actually remoting into a system pod -- I mentioned that for brevity in the steps to reproduce.  I have nginx deployed and that didn't work.

soooo as a "workaround"? i did just realize that if i do either of the following it starts working:

1. enter junk command 3 times (`asdf + ENTER`)
2. spam enter 6 times

note the quantities above -- they are consistent across different pods
![kube_terminal](https://user-images.githubusercontent.com/19917677/52561811-17efaf00-2dc2-11e9-978c-c87ade147ffa.gif)

**Comment by itowlson** [comment]
That's super weird.  Clearly the `exec` is working and creating the `bash` session on the pod.  But somehow the terminal isn't getting you to the shell prompt without a bit of help.  I've not seen this before and am not sure how to diagnose it.  We're not doing anything weird in the way of creating the terminal that I know of...

**Comment by cmeyertons** [comment]
yeah super weird -- now that i have a workaround, I probably wouldn't have even gone to the effort of creating an issue...

i'm fine w/ you closing this as others can still try to navigate to it
